src/main/xar-resources/data/newrangeindex/newrangeindex.xml

```diff
@@ -49,9 +49,11 @@
             string functions like <code>fn:contains</code>, <code>fn:starts-with</code>,
             <code>fn:ends-with</code>. </para>
         <note>
-            <para> <code>fn:matches</code> is currently not supported due to limitations in Lucene's
-                regular expression handling. If you require fn:matches a lot, consider using the
-                <link xlink:href="oldrangeindex">old range index</link>.</para>
+            <para><code>fn:matches</code> is currently not supported due to limitations in Lucene's
+                regular expression handling. Alternatives include the new <code>range:matches</code>
+                function (which <emphasis>is</emphasis> supported with the new range index, although subject to
+                <link condition="_blank" xlink:href="http://lucene.apache.org/core/4_5_1/core/org/apache/lucene/util/automaton/RegExp.html">Lucene 
```

Comment:
@djbpitt The version of Lucene shipped with eXist-db is currently 4.10.4. Would you mind updating the link to: https://lucene.apache.org/core/4_10_4/core/org/apache/lucene/util/automaton/RegExp.html

Reply:
Done! I also saw the cheat-sheet you mention below, but I was reluctant to link to that in the docs because I also don't know whether it is accurate for eXist-db.